app.rb

```diff
@@ -0,0 +1,104 @@
+require 'sinatra'
+require './fileserve.rb'
+require 'erb'
+require 'pathname'
+#require 'dir'
+#require 'ood-appkit'
+
+set :erb, :layout => :"layout.html" # use ERB as template engine and keeps the top bar
+set :escape_html, true    # safety first!
+set :static, true   # in case I feel like making this modular, set this explicitly
+set :public_folder, './public'  # same thing
+set :views, './views'   # yeah
+
+# fileserve = FileServer.new
+# fileserve.run!
+# register Sinatra::FileAPI
+
+# get '/fs/*' do
+#   fileserve.call(env)
+# end
+
+# put '/fs/*' do
+#   fileserve.call(env)
+# end
+
+# before do
+#   @env = env
+# end
+
+# serve '/fs/*'
+
+# This might get dangerous, but it's the only way I can see to deliver the js files without Sinatra 404ing
+# get '/public/*' do
+#   send_file './public/' + params[:splat][0]
+# end
+
+get '' do
```

Comment:
What is the goal of this?

Reply:
To be more user-friendly, I made it that if a person happens to go to the root URL of the app, then it automatically redirects to /edit/ where it gives a listing of the root of the file system on the web server. But of course I can remove that functionality if that is what you want for the future of the app.